adapter.yaml

```diff
@@ -1,38 +1,31 @@
-# hostname of a target gNMI server
-host: 127.0.0.1
-# gNMI port
-port: 50061
-# gNMI server username for all gNMI requests
-username: admin
-# gNMI server password for all gNMI requests
-password: admin
-# insecure/certificate-based gNMI server mode for all requests
-insecure: true
+device_config:
+  # hostname of a target gNMI server
+  host: 127.0.0.1
+  # gNMI port
+  port: 50061
+  # gNMI server username for all gNMI requests
+  username: admin
+  # gNMI server password for all gNMI requests
+  password: admin
+  # insecure/certificate-based gNMI server mode for all requests
+  insecure: true
 
-# default_encoding - leave "empty" for no overload of default encoding for Get requests,
-# use standard values otherwise - JSON|BYTES|PROTO|ASCII|JSON_IETF
-default_encoding: # "None" for no overload of default encoding for Get requests
-
-# configuration of gNMI library - whether to disable internal implementation logs
-enable_extra_logs: False
+lib_config:
+  # whether to enable internal implementation logs
+  enable_extra_logs: false
+  # GetRequest Encoding parameter if not set by test-case
+  default_encoding:
```

Comment:
I'm afraid this is not a `lib_config`, some devices prefer to work with (or even work only with) particular encoding; so while some test might flag that as a bug, we perhaps should allow tests to continue with an encoding of device's choice.